src/components/milvusCookieConsent/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import CookieConsent from "react-cookie-consent";
+import * as styles from "./milvusCookieConsent.module.less";
+import { useLocation } from "@reach/router";
+import { initializeAndTrack } from "gatsby-plugin-gdpr-cookies";
+
+function isBrowser() {
+  return typeof window !== "undefined";
+}
+
+function getValue(key, defaultValue) {
+  return isBrowser() && window.localStorage.getItem(key)
+    ? JSON.parse(window.localStorage.getItem(key))
```

Comment:
JSON.parse will cause some error when param is not json, we may need to use try catch for JSON.parse